src/app/hooks/useClickTracker/index.test.jsx

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import { render, act } from '@testing-library/react';
+import useClickTracker from '.';
+import pidginData from './fixtureData/tori-51745682.json';
+import zhongwenData from './fixtureData/chinese-news-49631219-trad.json';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+jest.useFakeTimers();
```

Comment:
I don't think you need this unless you're using setTimeout

Reply:
Ah yeah forgot to remove this, nice spot 👍